full-moon/src/tokenizer/structs.rs

```diff
@@ -390,6 +417,10 @@ pub enum TokenKind {
     #[cfg(feature = "luau")]
     /// Some form of interpolated string
     InterpolatedString,
+
+    #[cfg(feature = "cfxlua")]
+    /// A single/multi line C-style comment, such as `/* comment */`
+    CStylComment,
```

Comment:
Typo, also we don't need to mention single/multi line, since it can always be multiline
```suggestion
    /// A C-style comment, such as `/* comment */`
    CStyleComment,
```